partials/footer-widget-4col.php

```diff
@@ -5,17 +5,21 @@
 </div>
 
 <div class="span3 widget-area" role="complementary">
-	<?php if ( ! dynamic_sidebar( 'footer-2' ) ) {
-		the_widget( 'largo_featured_widget', array(
-				'term' => 'footer-featured',
-				'title' => __('In Case You Missed It', 'largo'),
-				'widget_class' => 'default',
-				'num_posts' => 2,
-				'num_sentences' => 2,
-				'thumb' => 'before'
-			)
-		);
-	}
+	<?php
+/**
+ * @see https://github.com/INN/largo/issues/1467
+ */
+//    if ( ! dynamic_sidebar( 'footer-2' ) ) {
```

Comment:
Commenting out this specific line prevents the `footer-2` sidebar from displaying, since calling `dynamic_sidebar( 'footer-2' )` is what causes that sidebar to be displayed. It may be best to comment out only the lines within the conditional, as you did in `partials/footer-widget-3col-equal.php` in this PR.